aidants_connect_web/migrations/0027_rename_mandat_step4.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.0.5 on 2020-05-28 13:16
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('aidants_connect_web', '0026_rename_mandat_step3'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='connection',
+            name='autorisation',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='usagers', to='aidants_connect_web.Autorisation'),
```

Comment:
idem question que dans le step 3. D'ailleurs, est-ce que step 3 et 4 pourraient être squashés ?

Reply:
Alors là, c'est différent, c'est le pendant du `step1` en fait, c'est à dire le moment où on retransforme ce champ en `ForeignKey` pour le "réattacher" au modèle lié (et donc, en sous-main, pour recréer la contrainte en base), après l'avoir initialement transformé en `IntegerField` pour l'en dissocier.

Pour info, je n'avais jamais été amené à faire ce changement précis, et je me suis appuyé sur les différentes solutions proposées dans [ce post StackOverflow](https://stackoverflow.com/questions/25091130/django-migration-strategy-for-renaming-a-model-and-relationship-fields) :)

Reply:
Du coup, je préfère ne pas les squasher parce que ce sont des opérations fondamentalement différentes — mais techniquement je pense qu'on pourrait sans problème.